content/concepts/basics/04-search.md

```diff
@@ -7,11 +7,41 @@ metaDescription: "sensenet Search"
 # Search
 https://community.sensenet.com/docs/search-engine/
 https://wiki.sensenet.com/query_syntax.html
-# Full text search
+
+In sensenet one of the most important technical aspects is how we index content items and how we search for them. Especially because sensenet is a query based system which means every content, every list you see on the UI is there because a query found it. In this article we describe the search module and how developers can customize it.
```

Comment:
```suggestion
In sensenet one of the most important technical aspects is how we index content items and how we search for them. Especially because sensenet is a query based system which means every content, every list you see on the ui is there because a query found it. In this article we describe the search module and how developers can customize it.
```